R/utils-testopt.R

```diff
@@ -155,7 +155,7 @@ domain_sphere <- function(){
 #'
 #' @export
 test_optim <- function(optim, ...,
-                       opt_hparams = list(lr = 0.01),
+                       opt_hparams = NULL,
```

Comment:
```suggestion
                       opt_hparams = list(),
```

Then you wouldn't need to check if it's `NULL` or not below.